test/autorest/lrogroup/lros_test.go

```diff
@@ -776,7 +827,6 @@ func TestLROBeginPutAsyncSubResource(t *testing.T) {
 }
 
 func TestLROBeginPutNoHeaderInRetry(t *testing.T) {
-	t.Skip("problem with put flow")
```

Comment:
Cool that these now work.  Presumably it's due to the LRO rewrite from last year.